modin/core/dataframe/algebra/binary.py

```diff
@@ -58,6 +61,8 @@ def caller(query_compiler, other, broadcast=False, *args, **kwargs):
                 at the query compiler level, so this parameter is a hint that passed from a high level API.
             *args : args,
                 Arguments that will be passed to `func`.
+            dtypes : "copy" or None, default: None
```

Comment:
why not make this a bool?

Reply:
there's some potential logic for an ability to pass ready-made dtypes, but the support for this isn't fully complete... maybe I should finish this little feature, too, and maintain the dtypes of filtered dataframe here...

Reply:
I think it's okay to leave this as is for now.